webcompat/views.py

```diff
@@ -133,10 +135,14 @@ def index():
         return render_template('index.html', form=bug_form,
                                browser=browser_name)
     # Form submission.
-    elif request.method == 'POST' and bug_form.validate():
+    elif bug_form.validate_on_submit():
         # copy the form so we can add the full UA string to it.
         form = request.form.copy()
         form['ua_header'] = ua_header
+        # Do we have an image ready to be uploaded?
+        image = request.files['image']
+        if image:
+            form['image_upload'] = json.loads(upload()[0])
```

Comment:
Both here or in the upload method, we are not checking the size of the file (or maybe I haven't found). I would put a limit on this. Imagine someone trying to upload a 10 Mb image… just for fun.